.circleci/config.yml

```diff
@@ -23,12 +23,14 @@ jobs:
       browser:
         type: string
     executor: linux
+    environment:
+      NODE_ENV: development # Needed for dist folder
     steps:
       - checkout
       - restore_cache:
-          key: deps-{{ .Branch }}--<< parameters.node-version >>--{{ checksum "package.json" }}
+          key: deps-{{ .Branch }}--<< parameters.node-version >>--{{ checksum "package.json" }}--{{ checksum "karma.conf.js" }}
```

Comment:
Great catch, @scottbell . I've made a dummy issue to track your change https://github.com/nasa/openmct/issues/4560